lib/wibox/container/radialprogressbar.lua

```diff
@@ -81,15 +86,19 @@ function radialprogressbar:after_draw_children(_, cr, width, height)
     local wa = outline_workarea(self, width, height)
     cr:translate(wa.x, wa.y)
 
+    local radius =  self._private.radius or
+        beautiful.radialprogressbar_radius or (wa.height/2)
```

Comment:
```suggestion 
        beautiful.radialprogressbar_radius or (math.min(wa.height, wa.width)/2)
```

Reply:
OK you're right.

I was looking at `shape.lua` in which this change has been made. And in the `require` section just after the header, one can see:
```
local min      = math.min
```
though in the code, there's a lot of `math.min` and a single `min` use.
Also in this file, there's no
```
local math = require("math")
```

So should I:

- put a `local math = require("math")` in `radialprogressbar.lua`?
- change `math.min(w, h) / 2` in `min(w, h) / 2` that I previously wrote in `shape.lua`?

Reply:
Oh yes, you're right. Math is some sort of core library in lua. It's implicitly required by lua itself so we don't need to manually require it ourself... My bad. 

> though in the code, there's a lot of `math.min` and a single `min` use.

The code was written over the years by different contributors. I'm not surprised to see this inconsistency. It's actually not important. 

> So should I:
> 
> - put a `local math = require("math")` in `radialprogressbar.lua`?
> - change `math.min(w, h) / 2` in `min(w, h) / 2` that I previously wrote in `shape.lua`?

First option is not necessary as math is already available, we don't need to require it. 

I'd say the second option is a cleaner way to manage this function call, but there is no obligation here.